packages/rum-core/src/domain/view/viewCollection.ts

```diff
@@ -126,7 +132,9 @@ function processViewUpdate(
       toServerDuration as (duration: Duration) => ServerDuration
     )
   }
+
   return {
+    customerContext: isExperimentalFeatureEnabled(ExperimentalFeature.VIEW_SPECIFIC_CONTEXT) ? view.context : undefined,
```

Comment:
❓ question: Do we need to check the FF? ‏The context should already be undefined from the tracking no?

Reply:
Indeed, I removed it.